### ISSUE TITLE
Posibility to pass many rst file or glob pattern

### ISSUE THREAD

**Issue by pwilczynskiclearcode** [twolfson/restructuredtext-lint#16]
It would be awsome if I could pass pattern or list of patterns instead of one file only:

```
rst-lint *.rst
```
```
rst-lint *.rst docs/source/*rst
```
```
rst-lint CHANGES.rst README.rst
```

I guess help text will look like
```
usage: rst-lint [-h] [--format FORMAT] [--encoding ENCODING] [path..]
```

Using proper [nargs](https://docs.python.org/2.7/library/argparse.html#nargs) and [glob](https://docs.python.org/2/library/glob.html#glob.glob) should do it:
```
    parser.add_argument('filepath', nargs='*', type=str, help='File to lint')
```

**Comment by twolfson** [comment]
Supporting multiple files is a great idea. I don't think glob support is great since:

- Most shells already will handle glob as is
- Prevents us from linting files containing glob characters (e.g. a file named `a*.b`)

For `nargs`, I think we want `+` since a filepath is always required.

https://docs.python.org/3/library/argparse.html#nargs

Want to open a PR for this?

**Comment by pwilczynskiclearcode** [comment]
> Want to open a PR for this?

Well, I haven't coded anything yet in this matter. Hard to promise.

**Comment by twolfson** [comment]
I can't tell. Is that a yes or a no? You see to grasp what needs to be done quite well.

**Comment by pwilczynskiclearcode** [comment]
Ok then, yes.

**Comment by twolfson** [comment]
This has been resolved by #19 and patched/released in `0.12.0`.
